py/torch_tensorrt/dynamo/__init__.py

```diff
@@ -1,6 +1,3 @@
-import torch
-from packaging import version
-
-if version.parse(torch.__version__) >= version.parse("2.1.dev"):
-    from torch_tensorrt.dynamo import fx_ts_compat
-    from .backend import compile
+from ._settings import *
+from .compile import compile
+from .aten_tracer import trace
```

Comment:
Consider wrapping in `if version.parse(torch.__version__) >= version.parse("2.1.dev"):`, since certain functionality and imports could break with earlier versions

Reply:
you mean the aten_tracer ?